src/index.html

```diff
@@ -447,7 +457,7 @@ <h2 class="title title_fz16 contacts__title">Контакты</h2>
                     <div class="title title_fz14 contacts__text">Любым удобным для вас способом:</div>
                     <div class="contacts__social">
                         <a href="#" class="contacts__link">
-                            <img src="icons/social/facebook.svg" alt="Facebook">
+                            <img src="icons/social/Facebook turn.svg" alt="Facebook">
```

Comment:
Filename must be in camelCase, snake_case or dash-case.
E.g. `facebook-turn.svg`

Reply:
ok, understand)

Reply:
Just a reminder: `Facebook-turn.svg` and `facebook-turn.svg` are two diffetent files